binding/HarfBuzzSharp.Shared/Face.cs

```diff
@@ -24,6 +27,16 @@ public Face (Blob blob, int index)
 			Handle = HarfBuzzApi.hb_face_create (blob.Handle, index);
 		}
 
+		public Face (TableLoader tableLoader)
+			: this (IntPtr.Zero)
+		{
+			this.tableLoader = tableLoader;
+			tableLoadFunc = tableLoader.Load;
```

Comment:
I will need to confirm this will work on iOS in Release mode on a device... I think the callbacks need to be a static method. That is why there is that static method with the `MonoPInvokeCallback` attribute. I don't think iOS supports the `UnmanagedFunctionPointer` like all the other platforms - might be an AOT thing.

Reply:
Not sure how to implement this for iOS with static function. Maybe it works the same as with BlobReleaseDelegate. We can store the context as user data aswell.